Source/CFStream.c

```diff
@@ -428,14 +445,22 @@ CFWriteStreamFDCopyProperty (CFWriteStreamRef s, CFStringRef propertyName)
           CFWriteStreamSetError(s, EBADF);
           return NULL;
         }
-          
-      offset = lseek(stream->fd, 0, SEEK_CUR);
 
+#ifdef _WIN32
+      offset = SetFilePointer(stream->handle, 0, 0, FILE_CURRENT);
+      if (offset == INVALID_SET_FILE_POINTER)
+        {
+          CFWriteStreamSetError(s, GetLastError());
```

Comment:
I think it returns Windows error codes, so probably not. Unfortunately the [documentation for `SetFilePointer()`](https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-setfilepointer) doesn’t specify the possible error codes so that we could convert them. It just says:
> If the function fails, the return value is INVALID_SET_FILE_POINTER. To get extended error information, call [GetLastError](https://docs.microsoft.com/en-us/windows/desktop/api/errhandlingapi/nf-errhandlingapi-getlasterror).

Not sure if there’s a better way to handle this but I’m open to suggestions.